tests/test_rlp.py

```diff
@@ -0,0 +1,361 @@
+import json
+import os
+from typing import List, Sequence, Tuple, Union
+
+import pytest
+
+from eth1spec import rlp
+from eth1spec.eth_types import Bytes, Uint
+from eth1spec.rlp import RLP
+
+from .helpers import hex2bytes
+
+#
+# Tests for RLP decode
+#
+
+
+@pytest.mark.parametrize(
```

Comment:
To be honest, I think we should generally avoid parameterized tests when we write them ourselves. I'd rather tests be clear and descriptive, even if that means there is some repetition. Also, since this repo is targeted at a general audience I do want to try to avoid tooling-specific things where we ca. I think someone with passing python knowledge may not be able to quickly grok this format. I really like how tests are generally done in rust. @SamWilsn did a great job on the etk tests, might be a good inspiration: https://github.com/quilt/etk/blob/7b2a166554af21325da5fe0a20a97d33b931f906/etk-asm/src/parse/mod.rs#L181-L191

Reply:
Currently working on splitting these to individual tests wherever possible.